### PR TITLE
i18n(zh-cn): update `guides/markdown-content.mdx`

### PR DESCRIPTION
**Pull request by 100gle**
#### Description (required)

update `guides/markdown-content.mdx`

#### Related issues & labels (optional)

#5031

### CODE REVIEW

File: src/content/docs/zh-cn/guides/markdown-content.mdx [hunk 1]
```diff
@@ -112,27 +112,27 @@ title: My page of content
...
 
-例如，要防止 `<` 被解释为 HTML 元素的开始，可以写 `&lt;`。 或者，要防止 `{` 被解释为 MDX 中 JavaScript 表达式的开始，可以写 `&lcub;`。
+例如，要防止 `<` 被解释为 HTML 元素的开始，可以写 `&lt;`。或者，要防止 `{` 被解释为 MDX 中 JavaScript 表达式的开始，可以写 `&lcub;`。
 
 ## MDX 独有特性
 
 添加 Astro [MDX 集成](/zh-cn/guides/integrations-guide/mdx/)可以使用 JSX 变量、表达式和组件来增强你的 Markdown 编写体验。
 
-它还为标准 MDX 添加了额外的功能，包括对 [MDX 中的 Markdown 样式 frontmatter](https://mdxjs.com/guides/frontmatter/)的支持。 这允许你使用 Astro 的大多数内置 Markdown 功能，例如 [frontmatter `layout`](#frontmatter-layout) 属性。
+它还为标准 MDX 添加了额外的功能，包括对 [MDX 中的 Markdown 样式 frontmatter](https://mdxjs.com/guides/frontmatter/)的支持。这允许你使用 Astro 的大多数内置 Markdown 功能，例如 [frontmatter `layout`](#frontmatter-layout) 属性。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/markdown-content.mdx:129]:
```suggestion
它还为标准 MDX 添加了额外的功能，包括对 [MDX 中的 Markdown 样式 frontmatter](https://mdxjs.com/guides/frontmatter/) 的支持。这允许你使用 Astro 的大多数内置 Markdown 功能，例如 [frontmatter `layout`](#frontmatter-layout) 属性。
```

File: src/content/docs/zh-cn/guides/markdown-content.mdx [hunk 2]
```diff
@@ -143,7 +143,7 @@ export const title = '我的第一篇 MDX 文章'
 ```
 ### 在 MDX 中使用 Frontmatter 变量
 
-Astro MDX 集成默认支持在 MDX 中使用 frontmatter。 在 Markdown 文件中添加 frontmatter 属性，这些变量可在模板中、其 [`layout` 组件](#frontmatter-layout)中以及在 [导入文件](#导入-markdown)时以命名属性的形式访问。
+Astro MDX 集成默认支持在 MDX 中使用 frontmatter。在 Markdown 文件中添加 frontmatter 属性，这些变量可在模板中、其 [`layout` 组件](#frontmatter-layout)中以及在 [导入文件](#导入-markdown)时以命名属性的形式访问。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/markdown-content.mdx:146]:
```suggestion
Astro MDX 集成默认支持在 MDX 中使用 frontmatter。在 Markdown 文件中添加 frontmatter 属性，这些变量可在模板中、其 [`layout` 组件](#frontmatter-layout)中以及在[导入文件](#导入-markdown)时以命名属性的形式访问。
```

File: src/content/docs/zh-cn/guides/markdown-content.mdx [hunk 3]
```diff
@@ -296,10 +296,10 @@ const posts = await Astro.glob<Frontmatter>('../pages/post/*.md');
 - **`file`** - 绝对文件路径（例如 `/home/user/projects/.../file.md`）。
 - **`url`** - 如果是页面，则为页面的 URL（例如 `/zh-cn/guides/markdown-content`）。
 - **`frontmatter`** - 包含文件中指定的 YAML frontmatter 中的任何数据。
-- **`getHeadings`** - 一个异步函数，返回文件中所有标题（即 `h1 -> h6` 元素）的数组。 每个标题的 `slug` 对应于给定标题的生成的 ID，可用于锚链接。 此列表遵循类型：`{ depth: number; slug: string; text: string }[]`。
+- **`getHeadings`** - 一个异步函数，返回文件中所有标题（即 `h1 -> h6` 元素）的数组。每个标题的 `slug` 对应于给定标题的生成的 ID，可用于锚链接。此列表遵循类型：`{ depth: number; slug: string; text: string }[]`。
 - **`Content`** - 一个组件，返回文件的完整渲染内容。
 - **（仅 Markdown)`rawContent()`** - 一个函数，返回原始 Markdown 文档作为字符串。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/markdown-content.mdx:301]:
```suggestion
- **（仅 Markdown）`rawContent()`** - 一个函数，返回原始 Markdown 文档作为字符串。
```

File: src/content/docs/zh-cn/guides/markdown-content.mdx [hunk 4]
```diff
@@ -321,7 +321,7 @@ import {Content as PromoBanner} from '../components/promoBanner.md';
 
 与其将 Markdown/MDX 文件放在 'src/pages/' 目录中来创建页面路由，不如[动态路由](/zh-cn/core-concepts/routing/#动态路由)。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/markdown-content.mdx:322]:
```suggestion
与其将 Markdown/MDX 文件放在 `src/pages/` 目录中来创建页面路由，不如[动态路由](/zh-cn/core-concepts/routing/#动态路由)。
```

File: src/content/docs/zh-cn/guides/markdown-content.mdx [hunk 5]
```diff
@@ -386,15 +386,15 @@ import Heading from '../Heading.astro';
...
 
-你可以在 `astro.config.mjs` 中自定义 remark 解析 Markdown 的方式。 请参阅完整的 [Markdown 配置选项](/zh-cn/reference/configuration-reference/#markdown-选项)列表，。
+你可以在 `astro.config.mjs` 中自定义 remark 解析 Markdown 的方式。请参阅完整的 [Markdown 配置选项](/zh-cn/reference/configuration-reference/#markdown-选项)列表，。
 
 ### Markdown 插件
 
-Astro 支持添加第三方 [remark](https://github.com/remarkjs/remark) 和 [rehype](https://github.com/rehypejs/rehype) 插件来解析 Markdown 和 MDX。 这些插件允许你扩展你的 Markdown，以获得新的功能，例如 [自动生成目录](https://github.com/remarkjs/remark-toc)，[应用可访问的表情符号标签](https://github.com/florianeckerstorfer/remark-a11y-emoji)，以及 [为你的 Markdown 添加样式](/zh-cn/guides/styling/#markdown-样式)。
+Astro 支持添加第三方 [remark](https://github.com/remarkjs/remark) 和 [rehype](https://github.com/rehypejs/rehype) 插件来解析 Markdown 和 MDX。这些插件允许你扩展你的 Markdown，以获得新的功能，例如 [自动生成目录](https://github.com/remarkjs/remark-toc)，[应用可访问的表情符号标签](https://github.com/florianeckerstorfer/remark-a11y-emoji)，以及 [为你的 Markdown 添加样式](/zh-cn/guides/styling/#markdown-样式)。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/markdown-content.mdx:397]:
```suggestion
Astro 支持添加第三方 [remark](https://github.com/remarkjs/remark) 和 [rehype](https://github.com/rehypejs/rehype) 插件来解析 Markdown 和 MDX。这些插件允许你扩展你的 Markdown，以获得新的功能，例如 [自动生成目录](https://github.com/remarkjs/remark-toc)，[应用可访问的表情符号标签](https://github.com/florianeckerstorfer/remark-a11y-emoji)，以及[为你的 Markdown 添加样式](/zh-cn/guides/styling/#markdown-样式)。
```

File: src/content/docs/zh-cn/guides/markdown-content.mdx [hunk 6]
```diff
@@ -386,15 +386,15 @@ import Heading from '../Heading.astro';
 
 ## 配置 Markdown
 
-Astro 中的 Markdown 支持由 [remark](https://remark.js.org/) 提供，remark 是一个强大的解析和处理工具，拥有一个活跃的生态系统。 目前不支持其他 Markdown 解析器，如 Pandoc 和 markdown-it。
+Astro 中的 Markdown 支持由 [remark](https://remark.js.org/) 提供，remark 是一个强大的解析和处理工具，拥有一个活跃的生态系统。目前不支持其他 Markdown 解析器，如 Pandoc 和 markdown-it。
 
-Astro 默认应用 [GitHub 风格的 Markdown](https://github.com/remarkjs/remark-gfm) 和 [SmartyPants](https://github.com/silvenon/remark-smartypants) 插件。 这带来了一些便利，例如从文本生成可点击的链接，并格式化 [引用和 em-dashes](https://daringfireball.net/projects/smartypants/)。
+Astro 默认应用 [GitHub 风格的 Markdown](https://github.com/remarkjs/remark-gfm) 和 [SmartyPants](https://github.com/silvenon/remark-smartypants) 插件。这带来了一些便利，例如从文本生成可点击的链接，并格式化 [引用和 em-dashes](https://daringfireball.net/projects/smartypants/)。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/markdown-content.mdx:391]:
```suggestion
Astro 默认应用 [GitHub 风格的 Markdown](https://github.com/remarkjs/remark-gfm) 和 [SmartyPants](https://github.com/silvenon/remark-smartypants) 插件。这带来了一些便利，例如从文本生成可点击的链接，并格式化[引用和 em-dashes](https://daringfireball.net/projects/smartypants/)。
```

File: src/content/docs/zh-cn/guides/markdown-content.mdx [hunk 7]
```diff
@@ -414,13 +414,13 @@ export default defineConfig({
 });
 ```
   
-请注意，默认情况下，`remarkToc`需要页面上的 “ToC” 或 “Table of Contents” [标题](https://github.com/remarkjs/remark-toc#optionsheading)（不区分大小写）才能显示目录。
+请注意，默认情况下，`remarkToc`需要页面上的“ToC”或“Table of Contents” [标题](https://github.com/remarkjs/remark-toc#optionsheading)（不区分大小写）才能显示目录。
```
Comment (Genteure) [inline src/content/docs/zh-cn/guides/markdown-content.mdx:417]:
```suggestion
请注意，默认情况下，`remarkToc` 需要页面上的“ToC”或“Table of Contents”[标题](https://github.com/remarkjs/remark-toc#optionsheading)（不区分大小写）才能显示目录。
```

**Review by Genteure — APPROVED** [review]
LGTM

All suggested changes above are formatting changes, they are nice to have but optional, and not a blocker for this PR to be merged.

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work peeps 🙌

**Pull request merged**